src/cc/export/helpers.h

```diff
@@ -90,6 +90,14 @@ struct _name##_table_t { \
 __attribute__((section("maps/perf_array"))) \
 struct _name##_table_t _name
 
+// Identifier for current CPU used in perf_submit and perf_read
+// Prefer use BPF_F_CURRENT_CPU flag, falls back to call helper for older kernel
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(4, 8, 0)
```

Comment:
BTW, does anyone actually do "forward port"? i.e. they have higher Kernel version number but don't have all the corresponding patches? If people normally just back port newer changes to older version, I think this particular change is fine because they are just missing the optimization, nothing will break.